Makefile

```diff
@@ -5,27 +5,19 @@ SHELL := /bin/bash
 export PROJECT_DIR            = $(shell 'pwd')
 export PROJECT_NAME			  = $(shell basename ${PROJECT_DIR})
 
-# set docker image tag to branch name or "latest" if "master" or "main" branch
-GIT_BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
-IMG_TAG = ${GIT_BRANCH}
-ifeq ("main",${IMG_TAG})
-IMG_TAG = latest
-endif
-ifeq ("master",${IMG_TAG})
-IMG_TAG = latest
-endif
+# Version to apply to generated artifacts (for bundling/publishing)
+export VERSION ?= 0.0.1
```

Comment:
Do we want to set this version for whatever our next intended release is? i.e. 0.2.1 or 0.3.0? It is getting overwritten when we create the index after pushing to `main` for _that_ process, but we want to make sure that if it is used in the bundle content generation that the proper version is being used to inject into places like the CSV.

Reply:
It just seems to me like it might be confusing to have a version defined that is inaccurate/never planned to be updated or used. I think that we have protected ourselves everywhere for this, but it would make sense to add some note about it -- similar to @chrisahl 's comment in a different thread.

Reply:
Note added to Makefile.